ldms/src/sampler/zfs_leafvdevs/Makefile.am

```diff
@@ -0,0 +1,17 @@
+pkglib_LTLIBRARIES =
+dist_man1_MANS =
+
+AUTOMAKE_OPTIONS = subdir-objects
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@ \
+	      -I/usr/include/libspl \
+	      -I/usr/include/libzfs
+
+AM_LDFLAGS = @OVIS_LIB_ABS@
+COMMON_LIBADD = -lsampler_base -lldms -lovis_util -lcoll \
+		@LDFLAGS_GETTIME@
+
+libzfs_leafvdevs_la_SOURCES = zfs_leafvdevs.c ../zfs_common/zfs_common.c
```

Comment:
Please include "../zfs_common/zfs_common.h" in SOURCES for each of these Makefile.am files. Without that, nothing in automake/autoconf references the zfs_common.h, so it isn't appearing in the "make dist" tarball.